vpc/service/gc_v3.go

```diff
@@ -39,6 +39,51 @@ const (
 	associated = "associated"
 )
 
+// getAllRegionAccounts gets regions accounts of the trunk ENI ("accounts"), as well as the
+// branch ENI accounts
+func (vpcService *vpcService) getAllRegionAccounts(ctx context.Context) ([]keyedItem, error) {
+	tx, err := vpcService.db.BeginTx(ctx, &sql.TxOptions{
+		ReadOnly: true,
+	})
+	if err != nil {
+		logger.G(ctx).WithError(err).Error("Could not start database transaction")
+		return nil, err
+	}
+	defer func() {
+		_ = tx.Rollback()
+	}()
+	rows, err := tx.QueryContext(ctx, "SELECT region, account_id FROM accounts")
```

Comment:
Although this would no longer be in a transaction, you could call `getRegionAccounts` and append instead of duplicate